app/serializers/disability_claim_base_serializer.rb

```diff
@@ -30,4 +31,8 @@ def open
   def waiver_submitted
     object.data['waiver5103Submitted']
   end
+
+  def development_letter_sent
+    object.data['developmentLetterSent'] == 'Yes'
```

Comment:
Yes, but I also don't want them to go back and redo their work if the schema changes underneath them.